### PR TITLE
Problem: recipient type in sendToCosmos is confusing

### PR DESCRIPTION
**Pull request by yihuang**
Solution:
- add a sendToCronos method which accept recipient in address type

it's tested in cronos's integration test: https://github.com/crypto-org-chain/cronos/pull/127

### CODE REVIEW

**Comment by tomtau** [comment]
@FrancoCRO

**Comment by tomtau** [comment]
any conflicts with https://github.com/crypto-org-chain/gravity-bridge/pull/5 ?

**Comment by yihuang** [comment]
> any conflicts with #5 ?

I think no.

File: solidity/contracts/Gravity.sol [hunk 1]
```diff
@@ -521,11 +521,19 @@ contract Gravity is ReentrancyGuard {
...
+	) public nonReentrant {
+		sendToCosmos(_tokenContract, bytes32(uint256(uint160(_destination))), _amount);
+	}
+
 	function sendToCosmos(
 		address _tokenContract,
 		bytes32 _destination,
 		uint256 _amount
-	) public nonReentrant {
+	) private {
```
Comment (FrancoCRO) [inline solidity/contracts/Gravity.sol:536]: Can we open both function to be public so users can choose the one they're comfortable to?

Comment (yihuang) [reply solidity/contracts/Gravity.sol:536]: But we need to remove `nonReentrant`, does that make it less safe to use? no strong opinion on it.

Comment (yihuang) [reply solidity/contracts/Gravity.sol:536]: Or can we keep the `nonReentrant` on both `sendToCosmos` and `sendToCronos` somehow?

Comment (FrancoCRO) [reply solidity/contracts/Gravity.sol:536]: ahhh I just realised the "confusing" part is people want to send to `Cronos` but instead they need to call `sendToCosmos`? 😂 

If that's the case I think you're right, we can keep it this way

**Review by FrancoCRO — APPROVED** [review]

File: solidity/contracts/Gravity.sol (hunk 1 above)
Comment (yihuang) [reply solidity/contracts/Gravity.sol:536]: The most confusing part is in `sendToCosmos`, the `bytes32` type, when you send a 20bytes address to it, by default the client pad it with zeroes on the right side, which results in the wrong address on Cronos side, you need to explicitly pad it on the left side.

Comment (yihuang) [reply solidity/contracts/Gravity.sol:536]: I notice there's a new method `sendToCosmosEth` now, maybe we also need a variant that accept recipient in `address` type?

**Pull request merged**